test/orchestrator/application_test.rb

```diff
@@ -168,6 +168,37 @@ def test_deploy_versions
       Application.new.deploy_versions(language: language_slug, version_slugs: version_slugs)
     end
 
+    def test_deployed_versions
+      language_slug = "python"
+      response = {
+        status: {
+          status_code: 200,
+          message: "OK"
+        },
+        context: {
```

Comment:
I'd probably not include this `context` block, because we don't use it, and I'd rather not have the extra mental overhead of wondering if it's important when we come back to this test in 6 months.